main.py

```diff
@@ -144,6 +147,7 @@ def parse_args(args):
             "timeout": utils.get(args, int, 10, ["timeout"]),
             "host": utils.get(args, str, "127.0.0.1", ["host"]),
             "port": utils.get(args, int, 8000, ["port"]),
+            "use_subprocess": utils.get(args, bool, True, ["subprocess-agents"])
```

Comment:
Keep the param names the same to reduce confusion: isolate_agents=bool?

Reply:
Ping on the naming confusion - please keep the naming consistent regardless of what it is called